js/htmlRenderer.js

```diff
@@ -0,0 +1,55 @@
+const clearElements = (elements) => {
+  while (elements.firstChild) {
+    elements.removeChild(element.lastChild);
```

Comment:
Looks pretty similar to how React does it 👍  
https://github.com/facebook/react/blob/0203b6567c6fd6274866c853ef938241d24551ec/packages/react-dom/src/client/ReactDOMLegacy.js#L123

Reply:
Nice!